evopy/individual.py

```diff
@@ -42,3 +51,66 @@ def evaluate(self, fitness_function):
         self.fitness = fitness_function(self.genotype)
 
         return self.fitness
+
+    def _reproduce_single_variance(self):
+        """Create a single offspring individual from the set genotype and strategy parameters.
+
+        This function uses the single variance strategy.
+
+        :return: an individual which is the offspring of the current instance
+        """
+        new_genotype = self.genotype + \
+                       self.strategy_parameters[0] * self.random.randn(self.length)
+        scale_factor = self.random.randn() * np.sqrt(1 / (2 * self.length))
+        new_parameters = [max(self.strategy_parameters[0] * np.exp(scale_factor), self._EPSILON)]
+        return Individual(new_genotype, self.strategy, new_parameters)
+
+    def _reproduce_multiple_variance(self):
+        """Create a single offspring individual from the set genotype and strategy.
+
+        This function uses the multiple variance strategy.
+
+        :return: an individual which is the offspring of the current instance
+        """
+        new_genotype = self.genotype + [self.strategy_parameters[i] * self.random.randn()
+                                        for i in range(self.length)]
+        global_scale_factor = self.random.randn() * np.sqrt(1 / (2 * self.length))
+        scale_factors = [self.random.randn() * np.sqrt(1 / 2 * np.sqrt(self.length))
+                         for _ in range(self.length)]
+        new_parameters = [max(np.exp(global_scale_factor + scale_factors[i])
+                              * self.strategy_parameters[i], self._EPSILON)
+                          for i in range(self.length)]
+        return Individual(new_genotype, self.strategy, new_parameters)
+
+    # pylint: disable=C0103
```

Comment:
Can you use the textual ID here instead of the numeric one? Typically, when you run `pylint`, you get both a numeric one and a textual one (like `no-members`, off the top of my head).